dataprofiler/profilers/base_column_profilers.py

```diff
@@ -303,7 +303,7 @@ def _update_column_base_properties(self, profile: dict) -> None:
         :type profile: base data profile dict
         :return: None
         """
-        self.match_count += profile.pop("match_count")
+        self.match_count += int(profile.pop("match_count"))
```

Comment:
Cast added to adhere to type specification in numeric stats mixin attribute initialization (avoids setting to np.int64)